SolARService_MappingAndRelocFrontend/tests/SolARServiceTest_MappingAndRelocFrontend/SolARServiceTest_MappingAndRelocFrontend.cpp

```diff
@@ -201,6 +217,45 @@ int main(int argc, char* argv[])
         auto overlay3D = componentMgr->resolve<display::I3DOverlay>();
         LOG_INFO("Remote producer client: AR device component created");
 
+#ifdef IS_GT_SEQ
+        // load GT pose data
+        std::string pathToData = arDevice->bindTo<xpcf::IConfigurable>()->getProperty("pathToData")->getStringValue();
+        std::ifstream infile(pathToData + "/solar_to_world.txt");
+        std::string line;
+        if (!infile.is_open()) {
+            g_solar2world = Transform3Df::Identity();
+        }
+        else {
+            for (int r = 0; r < 4; r++) {
+                for (int c = 0; c < 4; c++) {
+                    std::getline(infile, line);
+                    g_solar2world(r, c) = std::stof(line);
+                }
+            }
+            infile.close();
+        }
+
+        for (int i = 0; i < NBR_GT; i++) {
```

Comment:
It's not great that values like NBR_GT are hard-coded.
It needs to be more generic. Currently, this test will only work with the expected test data: if we ever want to run it on another one (with more or less GT), it will fail/need adaptation.
We could have a file format with some meta in a header that contains these values for example, or use a JSON/YAML format with existing parser libraries.
Not in this PR though, needs to some design decisions.